src/Payum/Offline/Action/FillOrderDetailsAction.php

```diff
@@ -16,6 +16,10 @@ class FillOrderDetailsAction implements ActionInterface
     public function execute($request)
     {
         RequestNotSupportedException::assertSupports($this, $request);
+        
+        if($request->getToken()->getDetails()) {
+            return;
+        }
```

Comment:
the values from order should be forced and overwrite values in details, custom ones could be set like this

https://github.com/Payum/Offline/pull/2/files#r24068718